### ISSUE TITLE
Install of @file-storage/commin fails on Docker & Mac silicon because of sharp

### ISSUE THREAD

**Issue by pierrelemee** [googlicius/file-storage#18]
# Install of @file-storage/commin fails on Docker & Mac silicon because of sharp

When I add the dependency `@file-storage/common` using `yarn` it fails with this message:

```bash
error /app/node_modules/@file-storage/common/node_modules/sharp: Command failed.
Exit code: 1
Command: (node install/libvips && node install/dll-copy && prebuild-install) || (node install/can-compile && node-gyp rebuild && node install/dll-copy)
Arguments: 
Directory: /app/node_modules/@file-storage/common/node_modules/sharp
Output:
sharp: Installation error: Invalid Version: 1.2.4_git20230717
```

According to [this issue](https://github.com/lovell/sharp/issues/2794#issuecomment-903042193) this `sharp` problem has been fixed at version 0.29, but the `common` package still references `0.28`. 

Would it be possible to upgrade to _at least_ 0.29 ? I can open a pull request if you want.

Beside, why is `sharp`even required ? Image stats fetching could be done within an extra package like `@file-storage/image`. Also, maybe [extracting metadata using `node-imagemagick`](https://github.com/yourdeveloper/node-imagemagick#readmetadatapath-callbackerr-metadata) could be a solid alternative as it relies on top of `imagemagick` command that is pretty standard on debian-based distros. Anyway, these are just suggestions :)

However, and because this issue blocks my current development, I'm afraid I'll give a try [to `storage-abstraction` instead](https://github.com/tweedegolf/storage-abstraction) :/

**Comment by dangoceaneering** [comment]
You are right, image handling should not placed in the common package, I'll move it to somewhere alse, may be a plugin.

**Comment by pierrelemee** [comment]
Again, if you like I can propose a PR for `sharp` version bump 💡

**Comment by dangoceaneering** [comment]
> Again, if you like I can propose a PR for `sharp` version bump 💡

That's ok, thanks.
